src/java.base/share/classes/java/lang/constant/package-info.java

```diff
@@ -89,6 +89,11 @@
  * It is also suitable for describing {@code invokedynamic} call sites in bytecode
  * reading and writing APIs.
  *
+ * <p>Other members of this package are {@link java.lang.constant.ModuleDesc}
+ * and  {@link java.lang.constant.PackageDesc}. They represent module and package
```

Comment:
```suggestion
 * <p>Other members of this package are {@link ModuleDesc}
 * and  {@link PackageDesc}. They represent module and package
```

same package and so no need to qualify.